.github/workflows/ci.yml

```diff
@@ -53,6 +53,25 @@ jobs:
 
       - run: yarn test
 
+  ember-template-lint:
+    name: ember-template-lint@${{ matrix.etlVersion }}
+    needs: [test]
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        etlVersion: [^4.0.0, 5.0.0-alpha.0]
```

Comment:
Will update to `etlVersion: [^4.0.0, ^5.0.0]`, once there is a stable release.

Reply:
https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/pull/245